app/controllers/houses_controller.rb

```diff
@@ -0,0 +1,27 @@
+class HousesController < ApplicationController
+  def index
+  	@houses = House.all
+  end
+
+  def show
+  	@house = House.find(params[:id])
+  end
+
+  def edit
+  	@house = House.find(params[:id])
+
+  end
+  def new 
+  	@house = House.create!(params[:house])
```

Comment:
What does `params[:house]` give you? Would expect `@house = House.new` in the controller action for a `new` page.